src/chains/ethereum/ethereum/tests/api/eth/createAccessList.test.ts

```diff
@@ -0,0 +1,474 @@
+import assert from "assert";
+import { EthereumProvider } from "../../../src/provider";
+import getProvider from "../../helpers/getProvider";
+import compile, { CompileOutput } from "../../helpers/compile";
+import { join } from "path";
+import { BUFFER_EMPTY, Data, Quantity } from "@ganache/utils";
+import { Ethereum } from "../../../src/api-types";
+import { AccessList } from "@ganache/ethereum-transaction/src/access-lists";
+import { Address } from "@ganache/ethereum-address";
+import { CallError } from "@ganache/ethereum-utils";
+import { ERROR, VmError } from "@ethereumjs/vm/dist/exceptions";
+import { EVMResult } from "@ethereumjs/vm/dist/evm/evm";
+
+const encodeValue = (val: string) => {
+  return Data.toString(val, 32).slice(2);
+};
+
+async function deployContract(provider, from, code) {
+  await provider.send("eth_subscribe", ["newHeads"]);
+  const transactionHash = await provider.send("eth_sendTransaction", [
+    {
+      from,
+      data: code,
+      gas: "0xfffff"
+    }
+  ]);
+  await provider.once("message");
+
+  const receipt = await provider.send("eth_getTransactionReceipt", [
+    transactionHash
+  ]);
+
+  return receipt.contractAddress;
+}
+
+describe("api", () => {
+  describe("eth", () => {
+    describe("createAccessList", () => {
+      let contract: CompileOutput;
+      let provider: EthereumProvider;
+      let from, to, addr, encodedTo, encodedAddr: string;
```

Comment:
Missing some types here (which is also true in the existing `call.test.ts` tests this was copied from)